### ISSUE TITLE
About RioShiina Wordwrap 

### ISSUE THREAD

**Issue by Berthog** [Yggdrasill-Moe/Niflheim#21]
Hello, thank you very much for your tools for Bloody Rondo and Sorcery Jokers, it's really useful and works perfectly! 
I just have one question. i'm working right now in a translation for Bloody Rondo, but I'm having some problems with wordwrap/lines wrap. I have tried with <br> or \n in the scripts since that's the common for to call a wordwrapping in other engines, but it didn't worked for this one. I would really appreciate your help, thank you in advance.

**Comment by Darkness-TX** [comment]
？
<img width="2098" alt="qian" src="https://user-images.githubusercontent.com/22828117/124340993-df97d800-dbeb-11eb-949d-2cc294104da2.png">
<img width="1716" alt="hou" src="https://user-images.githubusercontent.com/22828117/124340998-e6264f80-dbeb-11eb-8d2f-bbe8a18bccba.png">

**Comment by Berthog** [comment]
Yeah, I just realized that and my question sounds really dumb loool. Thank you very much for the answer!!! 
![image](https://user-images.githubusercontent.com/84604636/124343828-2bb72e00-db94-11eb-814e-801d446b14ca.png)

**Comment by Berthog** [comment]
Sorry, I'm reopening this issue because there's some files with different extension that the tools can't decrypt/encrypt and I wonder if you can help me. 
The extension of that files are .SCN and are in the BR_S.WAR. I think there's where it save some configurations related to the text/font and other useful things. 
![image](https://user-images.githubusercontent.com/84604636/134449509-e9d55ccc-773b-40dd-bd73-9974114d507b.png)
Thanks in advance.

**Comment by Darkness-TX** [comment]
These files are not processed. In fact, they are not encrypted too. I remember there is nothing particularly important in these files.You can use the hex editor to search for the text you need
